main.py

```diff
@@ -22,8 +26,10 @@ def webhook(request):
                 processed = handle_payload(jsonData)
 
                 if processed is False:
+                    logger.error("[ERROR] Error processing the payload")
```

Comment:
For testing can you simply print different level logs with some test mesaage

Reply:
Okay.